Source/Eto.WinForms/Forms/Controls/LabelHandler.cs

```diff
@@ -239,12 +239,12 @@ protected override void OnPaint(swf.PaintEventArgs e)
 					{
 						var r = rect;
 						r.Height = position.Size.Height;
-						swf.TextRenderer.DrawText(e.Graphics, position.Text, Font, r, ForeColor, BackColor, textFormat);
+						swf.TextRenderer.DrawText(e.Graphics, position.Text, Font, r, ForeColor, textFormat);
 						rect.Y += r.Height;
 					}
 					return;
 				}
-				swf.TextRenderer.DrawText(e.Graphics, Text, Font, rect, ForeColor, BackColor, textFormat);
+				swf.TextRenderer.DrawText(e.Graphics, Text, Font, rect, ForeColor, textFormat);
```

Comment:
Transparent background caused problems and background is already handled by control painting (OnPaintBackrdound)